test/functional/services/test_subjects.ts

```diff
@@ -151,15 +155,23 @@ export function TestSubjectsProvider({ getService }: FtrProviderContext) {
       });
     }
 
-    public async setValue(selector: string, text: string): Promise<void> {
+    public async setValue(
+      selector: string,
+      text: string,
+      options: ClearOptions = { withKeyboard: false }
+    ): Promise<void> {
       return await retry.try(async () => {
         log.debug(`TestSubjects.setValue(${selector}, ${text})`);
         await this.click(selector);
         // in case the input element is actually a child of the testSubject, we
         // call clearValue() and type() on the element that is focused after
         // clicking on the testSubject
         const input = await find.activeElement();
-        await input.clearValue();
+        if (options.withKeyboard === true) {
+          await input.clearValueWithKeyboard();
```

Comment:
@pheyos @dmlemeshko do either of you think we might want to use this all the time? Seems like a more user-like interaction.

Reply:
@spalger I think it might be good to make this the standard, but I'm not sure if this could impact stability of other tests, that's why I didn't change the default behavior. But I'm happy to give it a try in a follow-up.

Reply:
@spalger I remember `clearValueWithKeyboard` was not working everywhere, but we still can put it as default and use `clearValue` directly when it is needed.

Reply:
Cool, no need to worry about it now then.